index.html

```diff
@@ -116,9 +131,11 @@
             <br /> <br />
             <strong>It was a great success!</strong> The shop changed how volunteers and refugees interacted - it became a place where everyone liked to go. To help other vulnerable communities, we had the dream to support many other humanitarian organizations with this system so that they could experience the same success.
         </div>
-    </section>
+    </div>
+    </div>
```

Comment:
I believe this line closes `<section>` in line 94. Either make it a section or change 94 to a div